### ISSUE TITLE
Export options

### ISSUE THREAD

**Issue by anderspeders** [NRGI/nrgi-country-dashboards#1]
As a user I would like: 
- export as jpeg and maybe also csv

**Comment by anderspeders** [comment]
Ideally the download option would feature download of data for the specific visualization. Or will this simply be a link to the CKAN-data store?

**Comment by markbrough** [comment]
Yes, the data will be stored and saved in downloadable PNG and CSV formats. We could provide a link to the CKAN data store but might make things more complicated for people.

**Comment by anderspeders** [comment]
Yes, agree that we should provide the data for the specific visualization, which is what users would intuitively expect. 

Additionally we could then provide a general link to the CKAN instance.

**Comment by anderspeders** [comment]
Can we make sure to add: 

Source <Name of sources>, <download options - CSV, JPEG>

I do not see it in the latest dashboard prototype.
